Create Juniper kit

keith-smiley-gravwell opened the pull request:
<!-- 

The title of this PR should have the form:  [TYPE]([ISSUE_NUMBER]): [CHANGELOG_MESSAGE]

- If [TYPE] is fix or feat, the CHANGELOG_MESSAGE will be included in customer-facing, release change logs. 
- Other [TYPE]s WILL NOT be included in release change logs. 
- Check out https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type for other type ideas, or invent your own. 

-->

This PR DOES NOT address any issues
Net new kit

ashnwade commented on juniper/MANIFEST:
```diff
@@ -0,0 +1,168 @@
+	"Name": "Juniper",
+	"Desc": "",
+	"Readme": "Juniper Readme",
+	"Version": 3,
```
If we are creating this as a new kit, should this be version 1?

mike-wade-gravwell commented:
1. It needs images for the kit.
2. All the objects need descriptions

Besides those two it LGTM

keith-smiley-gravwell commented:
> 1. It needs images for the kit.
> 2. It needs a readme
> 3. All the objects need descriptions
> 
> Besides those items it LGTM

- Images are stock for the time being...I've requested permission to use the Juniper logo, but haven't heard back yet
- Added README as README.md as well as in the MANIFEST
- Added descriptions for all the things

mike-wade-gravwell reviewed (approved):
LGTM! Readme looks great.
